js/_profile.js

```diff
@@ -1,4 +1,11 @@
 export default function darkMode() {
   const darkMode = document.querySelector('[data-js="darkmode"]');
```

Comment:
To be more explicit, I would recommend to add `Button` in this name. So that it is clear for other developers that the reference is to a clickable button.
```suggestion
  const darkModeButton = document.querySelector('[data-js="darkmode"]');
```

Reply:
@macfrei Thanks so much for the suggestion, will do so!